src/components/new-landing-page/hero.tsx

```diff
@@ -11,15 +11,26 @@ const phrases: ReadonlyArray<string> = [
   'Live Your Values',
 ]
 
-// Write a function that creates "wordGroups" from the phrases
-// Extra credit if it throws an error if you make a phrase with a # of words that isn't exactly equal to 3
-
-const wordGroups: AnimateWords = [
-  { color: 'human-blue', words: ['more', 'join', 'Boost', 'Meet', 'Learn', 'Use', 'Mentor', 'Live'] },
-  { color: 'human-pink', words: ['human', 'international', 'Your', 'Amazing', 'New', 'Latest', 'Diverse', 'Your'] },
-  { color: 'human-blue', words: ['internet', 'community', 'Resume', 'People', 'Skills', 'Technology', 'Community', 'Values'] },
-]
+const createWordGroups = function(phrasesArray: ReadonlyArray<string>): readonly object[] {
+  return phrasesArray.reduce((AnimateWordsArray, phrase, index) => {
+    const wordsArray = phrase.split(' ')
+    if (wordsArray.length !== 3) {
+      throw new Error (`The ${index}th phrase (${phrase}) need to be exact 3 words`)
```

Comment:
Let's fix this typo, it should say "exactly"

Reply:
> Let's fix this typo, it should say "exactly"

typos corrected.

Reply:
Got this TypeScript warning, I think it is related to the way I define wordGroups. 
![Screen Shot 2022-03-17 at 5 41 52 PM](https://user-images.githubusercontent.com/16155615/158899872-de3aa796-4de5-4bc1-8889-fa7bdbcf2290.png)